.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Continuous Integration
+
+on: [push]
+
+jobs:
+  test:
+    name: Test by running cookiecutter
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: 'orbyter-cookiecutter'
+      - name: Install cookiecutter
+        run: pip install cookiecutter
+      - name: run ls
+        run: ls
+      - name: Cut a cookie (repo)
+        run: python -m cookiecutter --no-input orbyter-cookiecutter project_name=foundation repo_name=foundation
+      - name: Build project containers (make dev-start)
+        working-directory: foundation
+        run: make dev-start
```

Comment:
For future work, could we very all the expected docker containers get spun up?

Reply:
Want to add an issue and resolve?

Reply:
#59